pixart.js

```diff
@@ -1 +1,12 @@
-
+$("#set-color").on("click", function(event){
+        event.preventDefault()
+        $(".brush").css("background-color", $("#color-field").val())
+});
+for(var i = 0; i < 8000; i++ ){
+    $("body").append("<div class='square'></div>")
+};
+$(".square").each(function(){
+      $(this).on("mouseover", function() {
+          $(this).css("background-color", $("#color-field").val())
+        })
+});
```

Comment:
what we're to happen if i set the color by clicking the above click event, then changed whats in the value of the input? When I mouseover the squares, which color will be presented, the one thats set or the one in the input box. Is this a bug or feature? How could we change this implementation in order to set the brush color as the one in the click event rather than a value of the inputbox